Add support for an "authors" collection

tosky opened the pull request:
Handled through the admin UI, it is primarily used
to assign authors to articles.
It may be extended to support additional use cases
(for example team members data).

CKroes97 commented on config.yml:
```diff
@@ -105,6 +106,23 @@ collections: # A list of collections the CMS should be able to edit
+  - name: "authors"
+    label: "Authors"
+    folder: "_authors/"
+    slug: "{{name}}"
```
Using the name as a slug is bad practice since people can have the same first and -last name

tosky replied:
Uh, right, I guess {{ title }} is enough?

CKroes97 commented on config.yml:
```diff
@@ -105,6 +106,23 @@ collections: # A list of collections the CMS should be able to edit
           - {label: "Image", name: "image", widget: "image", required: false}
           - {label: "Thumbnail for Partner Overview", name: "teaser", widget: "image", required: false}
           - {label: "Label (e.g. for credit)", name: "credit", widget: "string", required: false}
+  - name: "authors"
```
You might want to wrap this in an "{% if site.cms.has_members %}{% raw %}

//your code


{% endraw %}

tosky replied:
Shouldn't this be always enabled? Otherwise the link inside the "posts" collection can be broken

CKroes97 reviewed (commented):
I left two comments, looks good for the rest

tosky committed:
Add support for an "authors" collection

Handled through the admin UI, it is primarily used
to assign authors to articles.
It may be extended to support additional use cases
(for example team members data).
